docs/troubleshooting/network-troubleshooting.mdx

```diff
@@ -7,6 +7,20 @@ slug: /troubleshooting/network-troubleshooting
 
 # Network Troubleshooting
 
+With the introduction of Light Hotspots, the network requirements and recommendations have changed. Light Hotspots do not require the p2p activity to connect to each other, they now connect directly to Validators to receive Challenges and report their witness receipts.
+
+The Light Hotspot => Validator connection is a standard gRPC connection.  The default port Validators listen on is 8080, so Hotspots should be able to connect Outbound to that port. (Note Validator operators can choose whichever port they want)  
```

Comment:
```suggestion
The Light Hotspot → Validator connection is a standard gRPC connection.  The default port Validators listen on is 8080 so Hotspots should be able to connect Outbound to that port. 

Note: Validator operators can choose whichever port they want
```

Reply:
is there any action required by hotspot owners to connect to a validator? is there a way for them to check?